dfget/config/constants.go

```diff
@@ -106,6 +107,14 @@ const (
 	StrBytes = "bytes"
 )
 
+/* piece meta */
+const (
+	PieceHeadSize = 4
```

Comment:
Could you please add comments here to make it clear?

Reply:
done